src/constants/sfx.asm

```diff
@@ -2,278 +2,278 @@
+MUSIC_THUNDER_DRUM                      equ $2E
+MUSIC_MARIN_SING                        equ $2F
+MUSIC_MANBOS_MAMBO                      equ $30
+MUSIC_OVERWORLD_INTRO                   equ $31 ; After obtaining sword
+MUSIC_MR_WRITE_HOUSE                    equ $32
+MUSIC_ULRIRA                            equ $33
+MUSIC_TARIN_BEES                        equ $34
+MUSIC_FROG_SONG                         equ $35
+MUSIC_MONKEYS_BUILDING_BRIDGE           equ $36
+MUSIC_CHRISTINE_HOUSE                   equ $37
+MUSIC_TOTAKA_UNUSED                     equ $38
+MUSIC_TURTLE_ROCK_ENTRANCE_BOSS         equ $39
+MUSIC_FISHERMAN_UNDER_BRIDGE            equ $3A
+MUSIC_OBTAIN_ITEM_UNUSED                equ $3B
+MUSIC_FILE_SELECT_TOTAKA                equ $3C ; Accessible only in German DX and Japanese versions. Enter file name MOYSE/とたけけ
+MUSIC_ENDING                            equ $3D
+MUSIC_MOBLIN_HIDEOUT                    equ $3E
+MUSIC_ISLAND_DISAPPEAR                  equ $3F
+MUSIC_RICHARD_HOUSE                     equ $40
+MUSIC_EGG_BALLAD_DEFAULT                equ $41 ; Play without instrument #3
+MUSIC_EGG_BALLAD_BELL                   equ $42 ; Play with instrument #3
+MUSIC_EGG_BALLAD_HARP                   equ $43 ; Play with instruments #3-4
+MUSIC_EGG_BALLAD_MARIMBA                equ $44 ; Play with instruments #3-5
+MUSIC_EGG_BALLAD_TRIANGLE               equ $45 ; Play with instruments #3-6
+MUSIC_EGG_BALLAD_ORGAN                  equ $46 ; Play with instruments #3-7
+MUSIC_EGG_BALLAD_ALL                    equ $47 ; Play with instruments #3-8
+MUSIC_GHOST_HOUSE                       equ $48
+MUSIC_ACTIVE_POWER_UP                   equ $49
+MUSIC_LEARN_BALLAD                      equ $4A
+MUSIC_CATFISHS_MAW                      equ $4B
+MUSIC_OPEN_ANGLERS_TUNNEL               equ $4C
+MUSIC_MARIN_ON_BEACH                    equ $4D
+MUSIC_MARIN_BEACH_TALK                  equ $4E
+MUSIC_MARIN_UNUSED                      equ $4F
+MUSIC_MINIBOSS                          equ $50
+MUSIC_KANALET_CASTLE_DUPE               equ $51
+MUSIC_TAIL_CAVE_DUPE                    equ $52
+MUSIC_DREAM_SHRINE_DREAM                equ $53
+MUSIC_EAGLE_BOSS_TRANSITION             equ $54
+MUSIC_ROOSTER_REVIVAL                   equ $55
+MUSIC_L2_SWORD                          equ $56
+MUSIC_HENHOUSE                          equ $57
+MUSIC_FACE_SHRINE                       equ $58
+MUSIC_WIND_FISH                         equ $59
+MUSIC_TURTLE_ROCK                       equ $5A
+MUSIC_EAGLES_TOWER                      equ $5B
+MUSIC_EAGLE_BOSS_LOOP                   equ $5C ; Changes to $54 after dialog
+MUSIC_FINAL_BOSS_INTRO                  equ $5D
+MUSIC_BOSS_DEFEAT                       equ $5E
+MUSIC_FINAL_BOSS_DEFEAT                 equ $5F
+MUSIC_FILE_SELECT_ZELDA                 equ $60 ; Enter file name ZELDA
+MUSIC_COLOR_DUNGEON                     equ $61
+MUSIC_SILENCE                           equ $FF
```

Comment:
Original comment explained what this was, and also that it doesn't always work

Reply:
Yeah, this definitely warrants a comment but I don't really understand how it functions. Will need someone to look into it for clarification. I'll add something for now.